libafl/src/corpus/cached.rs

```diff
@@ -38,7 +38,9 @@ where
             self.load_input_into(&mut testcase.borrow_mut())?;
             let mut borrowed_num = 0;
             while self.cached_indexes.borrow().len() >= self.cache_max_len {
-                let removed = self.cached_indexes.borrow_mut().pop_front().unwrap();
+                let removed = self.cached_indexes.borrow_mut().pop_front().ok_or_else(|| {
```

Comment:
this will never happen. the thing we really need is to make `cache_max_len` to NonZeroUsize